### DISCUSSION TITLE
We're now using Github Discussions!

### DISCUSSION THREAD

**Discussion by marcelveldt** [music-assistant/support#380] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mariusthvdb** [comment]
Hey Marcel,

Congrats with the launch, it has been a much anticipated feature for HA, having an integrated music player looking this swell.

Longtime HA user here, with a pro musical career, so glad to have been able to help test this ever since beta 01 ;-)

Cheers,
Marius (Marijn in NL)

**Comment by erkr** [comment]
To make sure developers have the necessary information to resolve issues, we introduced an issue template for bug reports.
Having the discussions sections for FAQ and new features or ideas, also implies the issues are now for bug reporting only. When opening an issue the appropriate links will be there as a reminder: 
![image](https://user-images.githubusercontent.com/4363599/174584599-be6cd09c-badd-40ac-b2e2-e992fe170d41.png)

**Reply by Mariusthvdb** [reply]
well done! nice, and so much better than searching in Discord chats ;-)

**Reply by OzGav** [reply]
Just enabled the ability to mark a question as ANSWERED so we can identify discussions that still need help and quickly show those searching with a similar problem what the answer is.

**Comment by Bachigaloupe** [comment]
New to Home Assistant and saw Marcel present MA on YouTube. Thanks for the hard work on this project!!!

I also saw the Raspiaudio presentation. 

I like the idea of using WiFi for sound but that little board was only 3 watts. Didn't seem like the goal for audiophiles either other than perhaps WiFi can offer improved bandwidth. 

Are there other products a little less costly than Sonos to use with MA? As media players? I guess I mean a place where we can post about hardware solutions we find and share with others.

I am thinking Amazon won't come around to lift restrictions. 

I hope I am posting in the correct place. Very new here.

**Reply by richardwonka** [reply]
Have you looked at ikea’s sonos speakers? They are readily available also used for very attractive prices and work as well as sonos’ (the essential hardware in there _is_ sonos)

**Comment by Bachigaloupe** [comment]
Here I am again. Sorry, a bit excited about getting access to my ripped music. 

So can MA send audio to the audio output of my RPI4?

**Comment by marcelveldt** [comment]
You have a lot of options. The raspiaudio has a small internal amplifier to direct drive some small speakers but it is a DAC so you can feed the output to any other more powerful amplifier.

I suggest you to take a look at the squeezelite software which is cross platform. We have created the slimproto integration in HA to directly interface squeezebox players (and the software implementation squeezelite) without the need for the Squeezebox server. Music Assistant will be directly streaming to those players and you can even group players.

You can run squeezelite on your hardware such as a pi yourself or use a ready made distribution such as picoreplayer. I have built a few radios for my kids with just a Pi Zero running with an I2s amp running piCorePlayer.

There is even a squeezelite version created for the ESP32. The raspiaudio board shown in the video and the Muse speakers run that for example.

**Reply by Bachigaloupe** [reply]
Thanks I will look into Squeezebox. I did see the other boards from Raspiaudio but they will require me to build something that need amplification.  I recently recovered old MacBook music, all my CDs, hours of ripping my owned music AND the Amazon music service keeps dropping or stalling so I come back to tell you my current plan. As I am still very new to all this, perhaps you might tell me if these are stupid ideas because they just can’t work. 

So I have a Bose Sound dock with an Aux in. It would seem to me if I can use it with the output or the RPI4 with a hard wired cable. I have an amplified stereo speaker. 

Alternatively I also have a Bose Bluetooth speaker and will investigate Bose Connect repositories to see if I can get a media player to work with MA. 

Sorry if I am using GitHub incorrectly. Gaining access, moving and renaming audio files on my NAS has me feeling anxious to get to music that was somewhat lost to me. 

Music and audio (other than Alexa notifications) were not something I considered when I started with Home Assistant. Until I watched your presentation. So I am (internally) blaming your for any new hardware expenses I make with respect to audio. 

Thanks for all your hard work for something I wouldn’t have imagined I would want!

**Comment by papabak** [comment]
I have recently installed MA in HA on my Raspberry Pi. It looks good. It was easy to add my Spotify premium account. I'm of course connect to my home network.
When I want to play a song and choose my Denon AVR3000X receiver it is switch on properly and the user interface indicates the song is playing (||). But, there is no sound and the progress bar doensn't show any progress. What can I do to solve this? I didn't find a solution in the FAQ.

Thanks for your help
Dirk

**Reply by erkr** [reply]
Hi,
Please check that the media player provided by the Denon integration its able to play url's (so MA compliant). Here some instructions: https://github.com/music-assistant/hass-music-assistant/discussions/622

**Comment by papabak** [comment]
I had a better look at the MA interface when trying to play music from my Spotify premium account. It shows it tries to play Ogg Vorbis formatted music, which  my Denon AVR3000X does not support.
Where can the format be selected/specified? Which program selects the Ogg Vorbis? MA perhaps?
I can find the Denon integration can play wav mp3 flac and ... (I forgot the last format)

![image](https://user-images.githubusercontent.com/97854063/197053608-c1e590fb-494f-4f01-8403-aea9e2fdd69c.jpeg)

**Reply by erkr** [reply]
You find it if you open the mass device. Enable the hidden entity content type.

**Reply by papabak** [reply]
Perhaps I'm too much of a novice. I found this MA setting for my Denon receiver. It doesn't mention ogg vorbis as format. The receiver is perfectly able to play Spotify via  Airplay or DLNA through my network. I still do not understand why MA seems to install fine, but doesn't start to play music.

[image](https://user-images.githubusercontent.com/97854063/197276349-5bb3d816-38d1-4232-b500-4a0369ecbefd.png)

**Reply by erkr** [reply]
Try to select the different options, mp3 or flac, to see if one of them works. These are the formats used by MA. The ogg vorbis format is the source format, transcoded info either mp3 or flac.

**Comment by WJ4IoT** [comment]
Just recently started with HA and today tried Music Assistant.

Documentation is good; specific  the "If Home Assistant does not show, refresh your browser (cache)." remark was very helpful. I have two radios (an original Logitech Radio and a ZeroPi with a BerryHAT and the piCorePlayer installed) and these were already set to SYNC in LMS. Which gives me the first observation of "Music Assistant". Apart from those two devices I also get the SYNCed one with an average volume:

![image](https://user-images.githubusercontent.com/17813643/226365516-e3dd7e50-4e97-42ee-a7f5-65234203c771.png)

Not a big deal for sure.

One of the options I missed starting with HA two weeks ago were my (LMS) favorites. Music Assistant is able to tackle this because I can see my TuneIn favorites. 

My last two small observation are :
- Still have the media control card in my dashboard; when a radio station is selected from Media assistant the additional info is not displayed.
- Music Assistant becomes an option inside browse media of the media control card but does not seem to work (not that I want to do it this way)

**Reply by marcelveldt** [reply]
I suppose you are still running Music Assistant v1 (in HA 2023.2) and not the beta of Music Assistant 2.0

The behavior you are seeing with the "groups player" is a feature as it allows you to control group volume.
So that is on purpose and actually very helpful (imo).

MA is meant as replacement for LMS not to be used side by side, in fact the 2.0 version will not even allow that anymore.
Favorites is the whole concept of MA where you can mark any item as favorite (or in library) and you can filter on that.

The media browser part should work but maybe it broke in the v1 version, As you may have read we're not updating the v1 version anymore as all development focus goes to v2 now.

**Reply by WJ4IoT** [reply]
Ok, thanks for fast reply will install beta version.

**Comment by agdamsbo** [comment]
I have been using HA for several years. Recently changed from iOS to /e/os (would recommend), which meant I had to abandon our AirPlay2 baser home audio setup. MA has been like having a revelation. I am very impressed and very happy to have a system, that is even better than before. Kudos! 👏👏👏🥳

**Comment by rwrozelle** [comment]
Spent the last 15 minutes removing "favorite current song" buttons that showed up out of nowhere on my Overview.

**Comment by paulearley** [comment]
It has probably been asked / discussed before, but is there a way of tiling Media Player / Queue / Recents / Players on one screen? Having more than one section display at the same time results in only one section functioning. 

This is a marvelous addition to the HA community!

**Reply by OzGav** [reply]
I think you mean with HA cards? If so, you would need to look at the community extensions and see what you can do with those https://www.music-assistant.io/community-extensions/
